src/styles/_movement.scss

```diff
@@ -41,7 +41,8 @@
   }
 }
 
-.animate {
+.animate,
+%animate {
```

Comment:
where is `%animate` used? (and thanks for teaching me about [placeholder selectors](https://sass-lang.com/documentation/style-rules/placeholder-selectors/), new concept to me! 🧑‍🏫 )

Reply:
You have the original developer of the feature to thank :)
We currently only use it to fade out modals on close, but there have been plans to animate more parts of the UI.

Reply:
I was just expecting to see an `@extend %animate` along with this, but I don't, am I misunderstanding how `%animate` would be used?

Reply:
You're right! Animation is actually not there. I was using automated tools to do most of the SASS upgrade and haven't noticed how the animations disappeared. Fix incoming.